Project-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,2 +1,15 @@
 import React from 'react';
 import './Header.css';
+
+const HeaderTitle = () => {
+
+    return(
+    <div className = 'title'>
+        <h4>Lambda School </h4>
+        <p>@LambdaSchool</p>
+        <p>. 26 jan</p>
```

Comment:
Look up Moment.js - a quick import that allows the use of 'moment()': which allow a real-time time-stamp for your projects!